src/main/java/io/github/moulberry/notenoughupdates/profileviewer/DungeonPage.java

```diff
@@ -232,17 +245,26 @@ public void drawPage(int mouseX, int mouseY, float partialTicks) {
 				boolean hasTime = false;
 				if (timeF5 > 1000) {
```

Comment:
Not this PRs fault but we should prevent code duplication like this.

Reply:
No need to change this in the levels pr

Reply:
This PR increases the amount of code duplication drastically. When it's using the existing code already, you can fix it on the way.

Reply:
That makes no sense at all.